tests/pytests/unit/modules/test_aixpkg.py

```diff
@@ -0,0 +1,1545 @@
+import logging
+import os
+
+## import salt.modules.rpm_lowpkg as rpm
+## import salt.modules.yumpkg as yumpkg
+import salt.modules.aixpkg as aixpkg
+
+## import salt.modules.cmdmod as cmdmod
+import salt.modules.pkg_resource as pkg_resource
+
+## from salt.exceptions import CommandExecutionError, SaltInvocationError
+## from tests.support.mock import MagicMock, Mock, call, patch
+from salt.exceptions import CommandExecutionError
+from tests.support.mock import MagicMock, patch
+
+## import salt.utils.platform
+
+
+## import pathlib
+
+
+try:
+    import pytest
+except ImportError:
+    pytest = None
+
+log = logging.getLogger(__name__)
+
+
+## Output from lslpp -Lc which contains Filesets and RPMs installed on system
+# Package Name:Fileset:Level:State:PTF Id:Fix State:Type:Description:Destination Dir.:Uninstaller:Message Catalog:Message Set:Message Number:Parent:Automatic:EFIX Locked:Install Path:Build Date
+
+lslpp_out = [
```

Comment:
This needs to be a pytest fixture. We cannot have mutable objects at the module level (https://docs.saltproject.io/en/master/topics/development/tests/index.html#module-global-level-variables)

Reply:
fixed